packages/mui/src/components/breadcrumb/index.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { useBreadcrumb, useRefineContext } from "@pankod/refine-core";
+import {
+    Breadcrumbs as MuiBreadcrumbs,
+    BreadcrumbsProps as MuiBreadcrumbProps,
+    Typography,
+    Link,
+    LinkProps,
+} from "@mui/material";
+
+import { HomeOutlined } from "@mui/icons-material";
+
+export type BreadcrumbProps = {
+    breadcrumbProps?: MuiBreadcrumbProps;
+    linkProps?: LinkProps;
+    showHome?: boolean;
+    hideIcons?: boolean;
+};
+
+export const Breadcrumb: React.FC<BreadcrumbProps> = ({
```

Comment:
```suggestion
};

import { Link as RouterLink } from "@refine/router-provider"

const LinkRouter = (props: LinkRouterProps) => (
  <Link {...props} component={RouterLink} />
);

export const Breadcrumb: React.FC<BreadcrumbProps> = ({
```

I think, we should use MUI Link like above otherwise the redirect will be hard refresh.